src/app/frontend/zerostate/zerostate.html

```diff
@@ -14,9 +14,28 @@
 limitations under the License.
 -->
 
-<div layout="vertical" layout-fill>
-  <header>
-    Zero state page. <a ui-sref="microservicelist">Go to microservices page</a> or
-    <a ui-sref="deploy">deploy an app</a>.
-  </header>
+<div layout="row" layout-align="center center" layout-padding class="kd-zerostate-content">
+  <md-content layout layout-align="center">
+    <md-card flex="50" class="kd-zerostate-deploy-card">
+      <div layout="row" class="kd-zerostate-card-header" layout-align="center center">
+        <md-icon md-svg-icon="assets/images/kubernetes-logo.svg" class="kd-zerostate-card-logo"></md-icon>
+        <span class="md-padding" flex>The <b>Kubernetes Dashboard</b> lets you deploy, monitor and troubleshoot containerized apps and services</span>
```

Comment:
Please, apply 100 char limit to this file. I know that this is not written rule, but it should land in style guide soon.